src/content/en/updates/2018/06/feature-policy.md

```diff
@@ -0,0 +1,336 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Feature Policy allows developers to selectively enable, disable, and modify the behavior of certain APIs and features in the browser. It's like CSP, but for features! Shipped in Chrome 60.
+
+{# wf_updated_on: 2018-06-22 #}
+{# wf_published_on: 2018-06-22 #}
+{# wf_tags: ux,feature,chrome60,feature-policy #}
+{# wf_featured_image: /web/updates/images/generic/checklist.png #}
+{# wf_featured_snippet: Feature Policy allows developers to selectively enable, disable, and modify the behavior of certain APIs and features in the browser. It's like CSP, but for features! Shipped in Chrome 60. #}
+{# wf_blink_components: Blink>FeaturePolicy #}
+
+# Introduction to Feature Policy {: .page-title }
+
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+<style>
+figure {
+  text-align: center;
+}
+figcaption {
+  font-style: italic;
+}
+</style>
+
+#### TL;DR {: #tldr .hide-from-toc}
+
+[Feature Policy][spec] allows web developers to selectively enable, disable, and
+modify the behavior of certain APIs and web features in the browser. __It's like
+[CSP](/web/fundamentals/security/csp/) but instead of controlling security, it's
+controls features!__
+
+The feature policies themselves are little opt-in aggreements between developer
+and browser that can help foster our goals of building (and maintaining) high
+quality web apps.
+
+## Introduction {: #intro}
+
+Building for the web is a rocky adventure. It's hard enough to build a top-notch
+web app that nails performance and uses all the latest best practices. It's even
+harder to keep that experience great over time. As your project evolves,
+developers come on board, new features land, and the codebase grows. That
+Great Experience &trade; you once achieved may begin to deteriorate and UX
+starts to suffer! Feature Policy is designed to keep us on track.
+
+With Feature Policy, you **opt-in to a set of "policies"** for the browser to
+enforce on specific features used throughout your site. These policies can
+restrict what APIs the site can access or modify the browser's default
+behavior for certain features.
+
+Here are a few examples of the types of things you can do with Feature Policy:
+
+1. Change the default behavior of `autoplay` on mobile and 3rd party videos.
```

Comment:
Writing best practice: reserve ordinals for procedures. Use bullets for a collection.

Reply:
3rd > third